modules/openapi-generator/src/test/resources/3_0/python-experimental/petstore-with-fake-endpoints-models-for-testing-with-http-signature.yaml

```diff
@@ -572,6 +572,9 @@ paths:
       summary: To test "client" model
       description: To test "client" model
       operationId: testClientModel
+      security:
+        - api_key_query: []
+          api_key: []
```

Comment:
Is this missing a `-`?

Reply:
No, I'm checking usage of `Multiple API Keys`.